mmv1/templates/terraform/examples/cloud_identity_groups_dynamic.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_cloud_identity_group" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Under L32 in in terraform.yaml, we'll want to add an additional `Examples` entry that corresponds to this config: https://googlecloudplatform.github.io/magic-modules/docs/how-to/add-mmv1-test/#terraformyaml-metadata

Reply:
@rileykarson - I've made these changes. Could we get another `/gcbrun`? 🙂